x-pack/plugins/snapshot_restore/public/application/lib/type_to_doc_url.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DocLinksStart } from 'kibana/public';
+import { RepositoryType } from '../../../common/types';
+import { REPOSITORY_TYPES } from '../../../common';
+
+export const getRepositoryTypeDocUrl = (docLinks: DocLinksStart, type: RepositoryType) => {
+  switch (type) {
+    case REPOSITORY_TYPES.fs:
+      return `${docLinks.links.snapshotRestore.registerSharedFileSystem}`;
```

Comment:
nit: The template literals are not necessary in this switch/case
```suggestion
      return docLinks.links.snapshotRestore.registerSharedFileSystem;
```